pkgs/development/compilers/gcc/common/configure-flags.nix

```
@@ -172,9 +174,8 @@ let
       then ["--enable-multilib" "--disable-libquadmath"]
       else ["--disable-multilib"])
     ++ lib.optional (!enableShared) "--disable-shared"
-    ++ [
-      (lib.enableFeature enablePlugin "plugin")
-    ]
+    ++ [ (lib.enableFeature enablePlugin "plugin") ]
+    ++ lib.optional (disableGdbPlugin) "--disable-libcc1"  # used only by GDB
```

```suggestion
    # Libcc1 is the GCC cc1 plugin for the GDB debugger which is only used by gdb
    ++ lib.optional disableGdbPlugin "--disable-libcc1"
```
I think the comment belongs more to here and I think the code is easier to understand this way

---

Absolutely.  Thanks for this!